How-to-find-the-commit-which-introduced-a-bug.md

```diff
@@ -0,0 +1,181 @@
+## Finding Bad Commits with the help of `git blame` and `git bisect`
+
+#### Before using both these commands:
+1. **Make sure to checkout the latest commit and see if the error exists.**
```

Comment:
Why do you need to bold everything? As mentioned before try to use bold only for important keywords, otherwise it's hard to read.

Reply:
Done